.github/workflows/dependabot-merge-prs.yml

```diff
@@ -0,0 +1,18 @@
+name: Auto-merge Dependabot PRs
```

Comment:
-1 on auto-merge, I'm pretty sure that won't work with branch protection anyway.

Reply:
removing